helm/csi-isilon/templates/controller.yaml

```diff
@@ -249,6 +255,37 @@ spec:
               mountPath: /isilon-configs
             - name: csi-isilon-config-params
               mountPath: /csi-isilon-config-params
+        {{- if hasKey .Values "authorization" }}
+        {{- if eq .Values.authorization.enabled true }}
+        - name: karavi-authorization-proxy
+          imagePullPolicy: Always
```

Comment:
Set imagePullPolicy to {{ .Values.imagePullPolicy }}

Reply:
Fixed.